code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -166,11 +151,6 @@
 		M.heal_organ_damage(3 * removed, 3 * removed)
 		M.adjustToxLoss(-3 * removed)
 
-/datum/reagent/tricordrazine/overdose(var/mob/living/carbon/M, var/alien)
```

Comment:
Inconsistent.
Tricrod has no effect on affect_blood on diona, but defaults to the standoard overdose effect, which effects diona.

Reply:
The same applies to a few others (dermaline, kelotane, dexalin are the ones I could find quickly)
All those do not affect diona in affect_blood, but affect diona in overdose